lib/Parser/Parse.cpp

```diff
@@ -11859,6 +11864,7 @@ HRESULT Parser::ParseSourceWithOffset(__out ParseNodeProg ** parseTree, LPCUTF8
     if (m_functionBody)
     {
         m_currDeferredStub = m_functionBody->GetDeferredStubs();
+        m_currDeferredStubCount = m_functionBody->GetNestedCount();
```

Comment:
Is the count of stubs necessarily equal to the nested count? I thought you only created stubs for potentially deferred functions.

Reply:
The count of stubs is strictly equal to the nested count. We create stubs for all nested functions. Original design expected all nested children to be deferred so it wasn't an issue at the time. Maybe worth improving this going forward but we will need more bookkeeping to keep track of the current deferred index.